src/controllers/management/finance.py

```diff
@@ -1,27 +1,29 @@
 from datetime import datetime
 import logging
+from src.mongodb.mongodb_models.patron_model import PatronDocument
 
-from src.models.entities.patrons.patron import Patron
 
-
-def calculate_bill(patron: Patron):
+def calculate_bill(patron: PatronDocument) -> float:
     """
-    Calculate the bill for a patron based on overdue library_items.
+    Calculate the bill for a patron based on overdue library items.
 
     Args:
-        patron (Patron): The patron for whom to calculate the bill.
+        patron (PatronDocument): The patron document for whom to calculate the bill.
 
     Returns:
         float: The calculated bill amount.
     """
-
     calculated_bill = 0
     check_date = datetime.now()
-    for patron_item_isbn, patron_item_due_date in patron.patron_items.items():
+
+    # Assuming `patron_items` in PatronDocument stores items as a dictionary where the key is the ISBN and value is the due date
```

Comment:
Why do you need this comment?
You created the types and decide how state is stored. You don't need to assume anything